locales/en/kafkaoverview.json

```diff
@@ -10,5 +10,16 @@
   "purchaseCardCallToActionButton": "Purchase now",
   "purchaseCardMainText": "Use Red Hat Marketplace to purchase a pay-as-you-go subscription for up to 25 streaming units. Purchasing through Red Hat Marketplace is only available in the United States and Canada.",
```

Comment:
New contents are provided for this section:
> In the United States or Canada, purchase a pay-as-you-go subscription for up to 25 streaming units through Red Hat Marketplace.

Reply:
New contents are also provided for the section with this string `contactSalesCardMainText` (a few lines up):
> Contact sales to get a pre-paid subscription or modify limits of your current subscription.